product/tests/test_services/test_item_creator.py

```
@@ -0,0 +1,159 @@
+from django.core.exceptions import ObjectDoesNotExist
+from django.http import Http404
+from django.test import TestCase
+
+from product.models.banner import Banner
+from product.models.category import Category
```

Ruff (0.8.2)

`product.models.category.Category` imported but unused

Remove unused import: `product.models.category.Category`

(F401)